### PR TITLE
Add custom classes to parent container

### PR DESCRIPTION
**Pull request by chtheis**
PR for issue #680

### CODE REVIEW

File: src/js/intlTelInput.js [hunk 1]
```diff
@@ -279,6 +281,10 @@ class Iti {
     let parentClass = 'intl-tel-input';
     if (this.options.allowDropdown) parentClass += ' allow-dropdown';
     if (this.options.separateDialCode) parentClass += ' separate-dial-code';
+    if (this.options.customParentClass) {
+      parentClass += ' ';
+      parentClass += this.options.customParentClass;
```
Comment (jackocnr) [inline src/js/intlTelInput.js]: This can be shortened to a single line e.g. `parentClass += ' ' + thing`

**Comment by jackocnr** [comment]
Hi there, thanks for putting this together, and especially for adding the test!

I actually don't like the name "parent" as it's a relative term and it's not clear what it's relating to. Please can we stick to `containerClass`. Also you will need to run `npm run build:js` to build the JS and commit the changes in /build/js and then the tests should pass. Thanks!

**Comment by chtheis** [comment]
Am 13.01.2019 um 18:54 schrieb jackocnr:
> *@jackocnr* commented on this pull request.
> 
> ------------------------------------------------------------------------
> 
> In src/js/intlTelInput.js 
> <https://github.com/jackocnr/intl-tel-input/pull/854#discussion_r247356173>:
> 
>> @@ -279,6 +281,10 @@ class Iti {
>       let parentClass = 'intl-tel-input';
>       if (this.options.allowDropdown) parentClass += ' allow-dropdown';
>       if (this.options.separateDialCode) parentClass += ' separate-dial-code';
> +    if (this.options.customParentClass) {
> +      parentClass += ' ';
> +      parentClass += this.options.customParentClass;
> 
> This can be shortened to a single line e.g. |parentClass += ' ' + thing|

Yes, but then lint (or whatever it is called here) was complaining 
loudly :(  I think it was even an error and not only a warning.

**Comment by chtheis** [comment]
Am 13.01.2019 um 18:55 schrieb jackocnr:
> Hi there, thanks for putting this together, and especially for adding 
> the test!
> 
> I actually don't like the name "parent" as it's a relative term and it's 
> not clear what it's relating to. Please can we stick to 
> |containerClass|. Also you will need to run |npm run build:js| to build 
> the JS and commit the changes in /build/js and then the tests should 
> pass. Thanks!

Done

**Comment by jackocnr** [comment]
Thanks!

**Pull request merged**